Bottom: a906fbde18e11e33ec3e9ca2debdeedfd23aa656
Top:    e43fb2f70123cb3c8ce8db14e418087db4f151f0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 01:49:52 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
